external/model-preparation-algorithm/tests/ote_cli/test_seg_cls_il.py

```diff
@@ -106,7 +106,6 @@ def test_ote_demo_openvino(self, template):
     @e2e_pytest_component
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_ote_deploy_openvino(self, template):
-        pytest.xfail("Known issue CVS-84981")
```

Comment:
What does this have to do with this PR? Is it related to OTE SDK update?

Reply:
yes, the segmentation deployment code was not working due to the old OTE SDK version. It's fixed after upgrading OTE SDK.